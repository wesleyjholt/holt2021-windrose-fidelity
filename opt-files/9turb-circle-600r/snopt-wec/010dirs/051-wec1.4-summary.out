
 ==============================
 S N O P T  7.7.3    (Dec 2019)
 ==============================
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.4-print.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.4-summary.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-

 SNMEMA EXIT 100 -- finished successfully
 SNMEMA INFO 104 -- memory requirements estimated



 Nonlinear constraints      45     Linear constraints       1
 Nonlinear variables        18     Linear variables         0
 Jacobian  variables        18     Objective variables     18
 Total constraints          46     Total variables         18



 The user has defined     828   out of     828   first  derivatives


 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0     13               1 (3.2E-12) 3.0E-01 -3.7222598E+03     13           r
     1      1  1.0E+00      2 (4.1E-09) 2.9E-01 -3.7224533E+03     13           r
     2      1  4.4E-01      3 (1.7E-07) 2.8E-01 -3.7236295E+03     13         s  l
     3      2  5.5E-01      5  2.3E-04  4.7E-01 -3.7420022E+03     14
     4      4  1.0E+00      6  1.0E-04  4.9E-01 -3.7613823E+03     13
     5      5  1.0E-01      8  8.9E-05  5.0E-01 -3.7719074E+03     15
     6      2  2.5E-01     10  6.7E-05  5.0E-01 -3.7855953E+03     14
     7      5  3.8E-01     12  1.0E-04  6.2E-01 -3.8066295E+03     14
     8      1  1.0E+00     13  3.4E-04  3.1E-01 -3.8461132E+03     14
     9      3  5.8E-01     15  6.7E-04  6.7E-01 -3.8672978E+03     14

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    10      3  1.0E+00     16  3.0E-04  3.3E-01 -3.8857567E+03     14
    11      3  1.0E+00     17  2.2E-03  3.0E-01 -3.8989028E+03     14
    12      1  1.0E+00     18  1.3E-04  1.8E-01 -3.9083135E+03     14
    13      1  1.0E+00     19  2.5E-04  1.5E-01 -3.9128801E+03     14
    14      1  1.0E+00     20  3.2E-04  1.2E-01 -3.9188338E+03     14
    15      1  1.0E+00     21  2.1E-04  6.7E-02 -3.9203190E+03     14
    16      1  1.0E+00     22  1.2E-05  3.2E-02 -3.9206753E+03     14
    17      1  1.0E+00     23  1.8E-06  2.9E-02 -3.9207501E+03     14
    18      1  1.0E+00     24 (3.4E-07) 2.9E-02 -3.9207813E+03     14
    19      1  1.0E+00     25  1.3E-06  1.9E-02 -3.9208097E+03     14

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    20      1  1.0E+00     26  1.8E-06  1.1E-02 -3.9208251E+03     14
    21      1  1.0E+00     27 (1.7E-07)(3.1E-03)-3.9208303E+03     14

 SNOPTA EXIT   0 -- finished successfully
 SNOPTA INFO   1 -- optimality conditions satisfied

 Problem name                 opt prob
 No. of iterations                  53   Objective           -3.9208304307E+03
 No. of major iterations            21   Linear    obj. term  0.0000000000E+00
 No. of infeasibilities              4   Sum of infeas        1.2790669632E-04
 Penalty parameter           0.000E+00   Nonlinear obj. term -3.9208304307E+03
 User function calls (total)        27
 No. of superbasics                 14   No. of basic nonlinears             4
 No. of degenerate steps             0   Percentage                       0.00
 Max x                      16 6.0E+02   Max pi                     43 1.1E+00
 Max Primal infeas          62 1.0E-04   Max Dual infeas            16 1.1E+00
 Nonlinear constraint violn    1.0E-04



 Solution printed on file  18

 Time for MPS input                             0.00 seconds
 Time for solving problem                       3.38 seconds
 Time for solution output                       0.00 seconds
 Time for constraint functions                  3.38 seconds
 Time for objective function                    0.00 seconds
