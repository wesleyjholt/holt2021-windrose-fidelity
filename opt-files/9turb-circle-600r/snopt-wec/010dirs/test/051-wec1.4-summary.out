
 ==============================
 S N O P T  7.7.3    (Dec 2019)
 ==============================
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.4-print.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.4-summary.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-

 SNMEMA EXIT 100 -- finished successfully
 SNMEMA INFO 104 -- memory requirements estimated



 Nonlinear constraints      45     Linear constraints       1
 Nonlinear variables        18     Linear variables         0
 Jacobian  variables        18     Objective variables     18
 Total constraints          46     Total variables         18



 The user has defined     828   out of     828   first  derivatives


 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0     11               1  8.1E-04  7.7E-01 -3.5154435E+03     11           r
     1      2  1.0E+00      2 (4.6E-08) 4.6E-01 -3.5159359E+03     12 3.4E+00   r
     2      3  6.0E-01      4 (6.1E-07) 5.5E-01 -3.5180432E+03     10 3.4E+00 sm l
     3      5  1.0E+00      5  7.6E-03  3.7E-01 -3.5450516E+03     14 3.4E+00
     4      3  1.0E+00      6  6.6E-04  4.2E-01 -3.5835134E+03     14 3.4E+00
     5      3  4.2E-01      8  3.3E-04  2.2E-01 -3.6173654E+03     14 3.4E+00
     6      2  1.0E+00      9  1.0E-04  2.0E-01 -3.6286493E+03     13 3.4E+00
     7      2  1.0E+00     10  1.8E-03  3.8E-01 -3.6434925E+03     12 3.4E+00
     8      1  1.0E+00     11  2.6E-04  4.8E-01 -3.6529141E+03     12 3.4E+00
     9      2  1.0E+00     12  1.4E-03  4.3E-01 -3.6721713E+03     11 3.4E+00

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    10      1  1.0E+00     13  1.9E-03  2.7E-01 -3.6965704E+03     11 3.4E+00
    11      1  1.0E+00     14  4.4E-04  2.9E-01 -3.7046339E+03     11 3.4E+00
    12      1  1.0E+00     15  1.1E-03  2.3E-01 -3.7239807E+03     11 3.4E+00
    13      1  6.6E-01     17  7.6E-04  2.5E-01 -3.7328987E+03     11 3.4E+00
    14      1  3.2E-01     20  5.2E-04  2.4E-01 -3.7351472E+03     11 3.4E+00
    15      1  2.4E-01     22  4.0E-04  3.0E-01 -3.7385300E+03     11 3.4E+00
    16      2  1.7E-03     32  4.0E-04  3.2E-01 -3.7385622E+03     10 3.4E+00
    17      4  2.6E-05     42  4.0E-04  1.7E+00 -3.7385644E+03     11 3.4E+00
    18      5  1.3E-01     44  4.0E-04  3.2E-01 -3.7453001E+03     11 3.4E+00
    19      2  1.0E+00     45  8.1E-05  2.5E-01 -3.7490763E+03     11 3.4E+00

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    20      2  1.0E+00     46  4.4E-04  2.2E-01 -3.7510664E+03     11 3.4E+00
    21      2  1.0E+00     47  8.1E-05  2.2E-01 -3.7530752E+03     11 3.4E+00
    22      1  1.0E+00     48  1.8E-05  2.2E-01 -3.7535675E+03     11 3.4E+00
    23      2  1.0E+00     49  3.8E-06  2.2E-01 -3.7537421E+03     11 3.4E+00
    24      2  1.0E+00     50  3.5E-06  2.1E-01 -3.7538794E+03     11 3.4E+00
    25      2  1.0E+00     51  2.5E-05  2.3E-01 -3.7541227E+03     11 3.4E+00
    26      2  1.0E+00     52  3.3E-05  2.4E-01 -3.7543918E+03     11 3.4E+00
    27      2  1.0E+00     53  1.6E-05  2.3E-01 -3.7545668E+03     11 3.4E+00
    28      2  1.0E+00     54  1.1E-06  2.2E-01 -3.7546835E+03     11 3.4E+00
    29      2  1.0E+00     55  2.7E-06  2.3E-01 -3.7548206E+03     11 3.4E+00

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    30      2  1.0E+00     56  1.2E-06  2.3E-01 -3.7549100E+03     11 3.4E+00
    31      2  1.0E+00     57  6.5E-06  2.5E-01 -3.7550830E+03     11 3.4E+00
    32      2  1.0E+00     58  3.1E-06  2.7E-01 -3.7551780E+03     11 3.4E+00
    33      2  2.3E-02     60  3.1E-06  2.3E-01 -3.7551905E+03     11 3.4E+00
    34      2  1.6E-02     62  3.0E-06  2.4E-01 -3.7551945E+03     11 3.4E+00
    35      1  1.0E+00     63  1.2E-06  2.4E-01 -3.7552297E+03     11 3.4E+00
    36      1  1.0E+00     64  1.6E-06  2.4E-01 -3.7552474E+03     11 3.4E+00
    37      1  1.0E+00     65 (1.3E-07) 2.4E-01 -3.7552524E+03     11 3.4E+00
    38      2  1.0E+00     66  1.1E-06  2.4E-01 -3.7552683E+03     11 3.4E+00
    39      1  6.8E-01     68  6.2E-06  2.3E-01 -3.7552885E+03     11 3.4E+00

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    40      2  1.0E+00     69 (3.8E-07) 2.2E-01 -3.7553033E+03     11 3.4E+00
    41      2  1.3E-01     73 (7.3E-07) 2.5E-01 -3.7553111E+03     11 3.4E+00
    42      1  1.0E+00     74  4.9E-06  2.5E-01 -3.7553301E+03     11 3.4E+00
    43      1  1.0E+00     75 (3.0E-07) 2.5E-01 -3.7553353E+03     11 3.4E+00
    44      1  1.0E+00     76 (8.4E-08) 2.5E-01 -3.7553386E+03     11 3.4E+00
    45      1  1.0E+00     77 (2.6E-07) 2.5E-01 -3.7553418E+03     11 3.4E+00
    46      1  1.0E+00     78 (2.7E-07) 2.5E-01 -3.7553431E+03     11 3.4E+00
    47      2  4.0E-01     89 (1.8E-07) 2.4E-01 -3.7553437E+03     11 3.4E+00  m
    48      1  2.6E-04    100 (1.8E-07) 2.4E-01 -3.7553438E+03     11 3.4E+00
    49      2  6.4E-06    110 (1.8E-07) 2.4E-01 -3.7553438E+03     11 3.4E+00   R

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    50      1  1.0E+00    111 (9.6E-09) 2.4E-01 -3.7553446E+03     11 3.4E+00 s
    51      1  1.0E+00    112 (3.5E-07) 2.4E-01 -3.7553454E+03     11 3.4E+00
    52      1  1.0E+00    113 (2.0E-08) 2.4E-01 -3.7553469E+03     11 3.4E+00
    53      1  3.8E-01    116 (1.6E-08) 2.5E-01 -3.7553472E+03     11 3.4E+00
    54      1  8.3E-02    126 (1.8E-08) 2.5E-01 -3.7553475E+03     11 3.4E+00
    55      1  5.0E-04    136 (1.8E-08) 2.5E-01 -3.7553475E+03     11 3.4E+00
    56      1  1.9E-06    146 (1.8E-08) 2.6E-01 -3.7553475E+03     11 3.4E+00
    57      1  2.2E-08    156 (1.8E-08) 2.6E-01 -3.7553475E+03     11 3.4E+00
    58      1  2.1E-10    165 (1.8E-08) 2.7E-01 -3.7553475E+03     11 3.4E+00
    59      1  2.2E-11    171 (1.8E-08) 2.8E-01 -3.7553475E+03     11 3.4E+00
    59      1             175 (1.8E-08) 2.4E-01 -3.7553475E+03     11           R
    59      6             179 (1.8E-08) 1.4E-01 -3.7553475E+03      6           r

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    60      8  1.1E-13    189 (1.8E-08) 2.2E-01 -3.7553475E+03     11         s
    61      1  9.0E-02    193 (1.7E-07) 1.9E-01 -3.7557047E+03     11
    62      1  2.7E-02    203 (1.9E-07) 1.8E-01 -3.7557911E+03     11
    63      1  2.8E-04    213 (1.9E-07) 1.8E-01 -3.7557916E+03     11
    64      1  3.0E-07    223 (1.9E-07) 1.7E-01 -3.7557916E+03     11
    65      1  1.6E-08    233 (1.9E-07) 1.7E-01 -3.7557916E+03     11
    66      1  3.9E-10    243 (1.9E-07) 1.7E-01 -3.7557916E+03     11
    67      1  5.4E-11    252 (1.9E-07) 1.7E-01 -3.7557916E+03     11
    67      1             259 (1.9E-07) 1.8E-01 -3.7557916E+03     11           R
    67      1             265 (1.9E-07) 1.8E-01 -3.7557916E+03     11           r

 SNOPTA EXIT  40 -- terminated after numerical difficulties
 SNOPTA INFO  41 -- current point cannot be improved

 Problem name                 opt prob
 No. of iterations                 136   Objective           -3.7557908895E+03
 No. of major iterations            67   Linear    obj. term  0.0000000000E+00
 No. of infeasibilities              5   Sum of infeas        2.7067746339E-04
 Penalty parameter           0.000E+00   Nonlinear obj. term -3.7557908895E+03
 User function calls (total)       291
 No. of superbasics                 11   No. of basic nonlinears             7
 No. of degenerate steps             0   Percentage                       0.00
 Max x                       6 5.8E+02   Max pi                     46 1.0E+00
 Max Primal infeas          57 1.1E-04   Max Dual infeas             6 7.1E-01
 Nonlinear constraint violn    1.1E-04



 Solution printed on file  18

 Time for MPS input                             0.00 seconds
 Time for solving problem                      31.62 seconds
 Time for solution output                       0.00 seconds
 Time for constraint functions                 31.62 seconds
 Time for objective function                    0.00 seconds
